sal-service/src/main/java/org/ow2/proactive/sal/service/service/MonitoringService.java

```diff
@@ -69,8 +68,7 @@ public class MonitoringService {
      * @param emsDeploymentDefinition An EMS deployment definition
      * @return return 0 if the deployment task is properly added.
      */
-    @Transactional
-    public synchronized Integer addEmsDeployment(String sessionId, EmsDeploymentDefinition emsDeploymentDefinition)
+    public Integer addEmsDeployment(String sessionId, EmsDeploymentDefinition emsDeploymentDefinition)
```

Comment:
I believe you should keep the @Transactionnal as this method updates the DB in a loop

Reply:
Well I want to. But Transactional is making this to crush with a `JDBCConnectionException` error